.github/workflows/helm-chart-release.yaml

```diff
@@ -22,6 +22,4 @@ jobs:
       - name: Run chart-releaser
         uses: helm/chart-releaser-action@v1.1.0
         env:
-          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
can you rebase master? We shouldn't rollback these changes :)

Reply:
Done. Sorry forgot to fetch upstream...